docs/src/upgrade/release-3.8.x.asciidoc

```diff
@@ -830,6 +843,38 @@ g.V().repeat(both()).times(2).dedup()
 
 See: link:https://issues.apache.org/jira/browse/TINKERPOP-3192[TINKERPOP-3192]
 
+==== Modified limit() and range() Semantics in repeat()
```

Comment:
Not sure if we also want to highlight this in the Provider section, but we should add an entry into the `gremlin-semantics` doc on this.